pkg/apis/v1alpha5/suite_test.go

```diff
@@ -409,6 +409,62 @@ var _ = Describe("Validation", func() {
 				Expect(provisioner.Validate(ctx)).ToNot(Succeed())
 			})
 		})
+		Context("GCThresholdPercent", func() {
```

Comment:
Can you add an e2e test?

Reply:
https://github.com/aws/karpenter/blob/main/test/suites/integration/webhook_test.go

Reply:
For sure, should I add a new e2e test in this project similar to the webhook_test.go or during my changes in the main project add the e2e test there?